content/contributing.md

```diff
@@ -17,33 +17,33 @@ It is a best practice to have your commit message have a _summary line_, followe
 
 Remember that not all users use Chef in the same way or on the same operating systems as you, so it is helpful to be clear about your use case and change so they can understand it even when it doesn't apply to them.
 
-### Github and Pull Requests
+### GitHub and Pull Requests
 
-We don't require you to use Github, and we will even take patch diffs attached to tickets on the issue tracker. However Github has a lot of convenient features, such as being able to see a diff of changes between a pull request and the main repository quickly without downloading the branch.
+We don't require you to use GitHub, and we will even take patch diffs attached to tickets on the issue tracker. However GitHub has a lot of convenient features, such as being able to see a diff of changes between a pull request and the main repository quickly without downloading the branch.
 
 ## Functional and Unit Tests
 
-This cookbook is set up to run tests under [Test Kitchen](http://kitchen.ci/). It uses serverspec to run integration tests after the node has been converged to verify that the state of the node.
+Our cookbooks are set up to run tests under [Test Kitchen](http://kitchen.ci/). They use InSpec to run integration tests after the nodes have converged to verify that the system state..
 
 Test kitchen should run completely without exception using the default [baseboxes provided by Chef](http://chef.github.io/bento/). Because Test Kitchen creates VirtualBox machines and runs through every configuration in the .kitchen.yml file, it may take some time for these tests to complete.
```

Comment:
I think the whole area really needs to get rewritten. The goal here was to nudge it closer to reality. It should mention our complete test strategy not just local test